core/tests/protractor_desktop/userJourneys.js

```diff
@@ -242,6 +242,10 @@ describe('Site language', function() {
     // Spanish is still selected.
     var placeholder = await element(by.css('.protractor-test-float-form-input'))
       .getAttribute('placeholder');
+    await waitFor.visibilityOf(placeholder, 'Placeholder taking too long to appear');
+    await waitFor.textToBePresentInElement(
+      placeholder, 'Ingresa un número',
+      'Placeholder text taking too long to change from English to Spanish');
```

Comment:
In my investigation I found out that when the placehoder became visbile it still contained the english version of the text but after a few seconds the text automatically changed to Spanish.